rplugin/python3/denite/scantree.py

```diff
@@ -0,0 +1,48 @@
+# ============================================================================
+# FILE: scantree.py
+# AUTHOR: Jorge Rodrigues <skeept at gmail.com>
+# License: MIT license
+# ============================================================================
+
+import sys
+from os.path import basename, join
+from os import walk
+
+try:
+    from os import scandir
```

Comment:
It would be nice to comment

```
# 'scandir' has introduced at Python 3.5. See https://pypi.python.org/pypi/scandir
```

For users who do not know about `scandir`